modules/fundamental/src/advisory/model/details/advisory_vulnerability.rs

```diff
@@ -57,8 +49,11 @@ impl AdvisoryVulnerabilityHead {
             };
             Ok(AdvisoryVulnerabilityHead {
                 head,
-                severity: score.severity(),
-                score: score.value(),
+                severity: advisory
+                    .average_severity
+                    .map(|sev| sev.into())
+                    .unwrap_or(Severity::None),
+                score: advisory.average_score.unwrap_or(0.0),
```

Comment:
This is actually a bug: #1374 

Can we just change these to Options in `AdvisoryVulnerabilityHead` and kill 2 birds with 1 stone? :smile:

Reply:
Yes. It bothered me as well, but didn't want to change API on my own. I can include it as well.

Reply:
Not sure when this is getting merged, so went ahead and pushed #1591